src/common/components/core/dialog/DialogActions.ts

```diff
@@ -3,7 +3,10 @@ import MuiDialogActions from '@mui/material/DialogActions';
 
 const DialogActions = styled(MuiDialogActions)(({ theme }) => ({
```

Comment:
this is used in many places, are you sure about this?

Reply:
I am not sure what are you suggesting.